irods_specific_implementation/irods_rest_api_base.h

```diff
@@ -232,7 +232,7 @@ namespace irods::rest
 
         std::string decode_url(const std::string& _in) const
         {
-            trace("Decoding input [{}] ...", _in);
+            trace("Decoding input [{}] ...", _in); // TODO This prints incoming arguments! (e.g. Passwords!)
```

Comment:
perhaps this is why we left it as a trace message.

still, logging people's passwords is wrong. we should remove this line.

Reply:
left note on other PR... just comment this out, along with a note as to why - available for devs to turn it back on if helpful.   but can never write in production.

Reply:
left a comment about why it is disabled.